### ISSUE TITLE
Cannot find module './selectors' from 'index.js'

### ISSUE THREAD

**Issue by smddzcy** [ReactTraining/react-router#5645]
## Version
5.0.0-alpha.7

## Test Case
https://codesandbox.io/s/n55VljYk7

## Steps to reproduce
- Install the `5.0.0-alpha.7` version of `react-router-redux`
- Try to import and use the library

## Expected Behavior
Importing the library without any problems

## Actual Behavior
It throws an error like this:
```
Module not found: Error: Can't resolve './selectors' in '[some-path]/node_modules/react-router-redux'
```

**Comment by DiMoNTD** [comment]
I'm confirm, there is a problem

**Comment by 0xR** [comment]
Same here

**Comment by agionfriddo** [comment]
Same. 🙁

**Comment by pshrmn** [comment]
This has been fixed in #5638 and brought up in #5637, #5636, #5640 (please check existing issues/PRs before opening a new one :smile_cat:). A release will be happen, but for the time being you can stick to `alpha.6`.
